test/unit/auth/token-verifier.spec.ts

```diff
@@ -301,13 +291,34 @@ describe('FirebaseTokenVerifier', () => {
               verifyApiName: 'verifyToken()',
               jwtName: 'Important Token',
               shortName: 'token',
-              expiredErrorCode: invalidExpiredErrorCode as any,
-              errorCodeConfig: AUTH_ERROR_CODE_CONFIG,
+              errorCodeConfig: verifierAuth.ID_TOKEN_ERROR_CODE_CONFIG,
+              errorType: invalidErrorCodeTypes as any,
```

Comment:
Did you mean to pass `invalidErrorCodeType` here?

Reply:
Thanks! Fixed.